### PR TITLE
Update lf path on kakoune buffer change

### PR DESCRIPTION
**Pull request by EmmChriss**

### CODE REVIEW

**Comment by TeddyDD** [comment]
please rebase it on master and squash the commits. Short description of new option in README (as well as changelog update) would be nice addition

File: rc/lf.kak [hunk 1]
```diff
@@ -35,6 +36,17 @@ hook -group lf global GlobalSetOption 'lf_id=\d+' %{
     '
 }
 
+hook -group lf global GlobalSetOption 'lf_update_path=(true|false)' %{ evaluate-commands %sh{
+    case $kak_opt_lf_update_path in
+        true)
+            echo '
+            hook -group lf-update-path global WinDisplay .* %{
+                lf-send-command "select %val{buffile}"
```
Comment (TeddyDD) [inline rc/lf.kak:50]: first you need to send cd command (user can open file from different directory)

**Review by TeddyDD — CHANGES_REQUESTED** [review]

File: rc/lf.kak (hunk 1 above)
Comment (EmmChriss) [reply rc/lf.kak:50]: Sorry, force pushed, could you request again?

**Comment by TeddyDD** [comment]
I merged it in 09e77c4a45c139a4dd235b8ce82827c747bfd481 with some fixes.

Thank you very much!

**Pull request closed without merging**